### PR TITLE
stig.sql - Add brackets as delimiter for database name

### LINKED ISSUE #6571: Possible bug, or improvement in Get-DbaUserPermission
Hi guys good afternoon 

I've run this particular script against a server of a client where all the databases have the first character of the name a number as example 4PEOPLE. This script basically ignores all the databases on that server.

Example:
Get-DbaUserPermission -SqlInstance SQLDEV -Database 4PEOPLE -ExcludeSystemDatabase | Out-GridView

### PR DESCRIPTION
**Pull request by andreasjordan**
## Type of Change

 - [x] Bug fix (non-breaking change, fixes #6571 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
Enable non regular database names.

### Approach
Add brackets as delimiter for database name

### Commands to test
Rename a sample database to "4Test" and run the command.

### Screenshots
Sorry, takes to much time...

### Learning
https://docs.microsoft.com/en-us/sql/relational-databases/databases/database-identifiers

### CODE REVIEW

File: functions/Get-DbaUserPermission.ps1 [hunk 1]
```diff
@@ -217,7 +217,7 @@ function Get-DbaUserPermission {
 
                 $sqlFile = Join-Path -Path $script:PSModuleRoot -ChildPath "bin\stig.sql"
                 $sql = [System.IO.File]::ReadAllText("$sqlFile")
-                $sql = $sql.Replace("<TARGETDB>", $db.Name)
+                $sql = $sql.Replace("<TARGETDB>", ('[' + $db.Name + ']'))
```
Comment (wsmelton) [inline functions/Get-DbaUserPermission.ps1:220]: I would prefer to just fix the syntax in the stig.sql file since we own it.

Comment (andreasjordan) [reply functions/Get-DbaUserPermission.ps1:220]: Ok, I was not aware that this code is not third party but owned by dbatools. Give me some minutes...

**Comment by andreasjordan** [comment]
Ok, now the fix is implemented in stig.sql instead of Get-DbaUserPermission.ps1

**Comment by wsmelton** [comment]
Maybe the test for the backup history command will run properly this time...⚱️

**Comment by andreasjordan** [comment]
> Maybe the test for the backup history command will run properly this time...⚱️

I would not bet on that...

**Review by wsmelton — APPROVED** [review]

**Pull request merged**